ui/apps/platform/src/Components/PatternFly/CodeEditorDarkModeControl.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { CodeEditorControl } from '@patternfly/react-code-editor';
+import { MoonIcon, SunIcon } from '@patternfly/react-icons';
+
+export type CodeEditorDarkModeControlProps = {
+    isDarkMode: boolean;
+    onToggleDarkMode: () => void;
+};
+
+/**
+ * A PatternFly code editor control that toggles dark mode.
+ */
+function CodeEditorDarkModeControl({
+    isDarkMode,
+    onToggleDarkMode,
+}: CodeEditorDarkModeControlProps) {
```

Comment:
We are also doing this in another place where this common component could be leveraged:

- ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPoliciesYAML.tsx